### DISCUSSION TITLE
Is this still the same as cnvs/canvas repo?

### DISCUSSION THREAD

**Discussion by ReeceM** [austintoddj/canvas#938] [Troubleshooting] [answered]
Hi, so I may have been out the loop for a while with Canvas, but did something change?

The cnvs.io website is for sale, so that wasn't reassuring from a bookmark...

But I just want to know if this is the same one as before.

**Comment by ReeceM** [comment]
The fact it's stilled starred in my list, gives the idea that it was a vendor/username change over.

**Comment by austintoddj** [comment, marked as answer]
**This is the same project, yes.**

A few months back I transitioned the project back to my personal account (which is where it started years ago), and updated the main landing page to [trycanvas.app](https://trycanvas.app)

I dropped a 301 forwarding redirect onto the domain in GoDaddy, so it should properly drop you off in the right spot. Apologies for any inconvenience.

**Reply by ReeceM** [reply]
> This is the same project, yes.

Awesome 👍 

Glad that is so then.

And the 301 does work, I think that might help anyone who's upgrading from some of the older versions, that is what happened to myself.

**Reply by austintoddj** [reply]
If you wouldn't mind marking the above answer as the Solution, others who run into this should find the answer they're looking for quickly 😉 

Thanks!

**Reply by ReeceM** [reply]
> If you wouldn't mind marking the above answer as the Solution, others who run into this should find the answer they're looking for quickly 😉
> 
> Thanks!

I was actually looking for that the first time I replied, it doesn't seem to be showing anywhere...

From the docs [Marking Answers](https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#marking-a-comment-as-an-answer)

> You can mark a comment in the discussion as an answer to the discussion if a discussion is within a category that accepts answers. For more information, see "About discussions." 

So I am not sure if I asked this in a topic that has that on, if not I'll move it to the right one and mark it there?

**Reply by austintoddj** [reply]
Ah makes sense, this is a general discussion, not a question. No worries!

**Comment by austintoddj** [comment]
Closing as historical.

Yes — this is the same project, now under `austintoddj/canvas`. Current major is [v7.0.0](https://github.com/austintoddj/canvas/releases/tag/v7.0.0).
